src/index.html

```diff
@@ -4,9 +4,346 @@
+            <a class="nav__link" href="#tablets">
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main">
+      <section class="product-card" data-qa="card">
+        <img
+          class="product-card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        >
+        <h3 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        
+        <p class="product-card__code">
+          Product code: 195434
+        </p>
+        <div class="product-card__review">
+          <ul class="stars stars--4">
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+          </ul>
+          <p class="product-card__review-count">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-sum">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="product-card__button"
+          data-qa="card-hover"
+        >
+          Buy
+        </a>
+      </section>
+      <section class="product-card" data-qa="card">
+        <img
+          class="product-card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
+        <h3 class="product-card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h3>
+        <p class="product-card__code">
+          Product code: 195434
+        </p>
+        <div class="product-card__review">
+          <ul class="stars stars--4">
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+            <li class="stars__item"></li>
+          </ul>
+          <p class="product-card__review-count">Reviews: 5</p>
+        </div>
+        <div class="product-card__price">
+          <p class="product-card__price-title">Price:</p>
+          <p class="product-card__price-sum">$2,199</p>
+        </div>
+        <a
+          href="#"
+          class="product-card__button"
+          data-qa="hover"
+        >
+          Buy
+        </a>
+      </section>
+      <section class="product-card" data-qa="card">
+        <img
+          class="product-card__image"
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)">
```

Comment:
![image](https://github.com/mate-academy/layout_catalog/assets/54860641/30e155e2-a4cb-4c80-afb5-314232949ea8)
Fix this in all code.